pages/adminDashboard.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+
+const AdminNavbar: React.FunctionComponent = () => {
+  const [title, setTitle] = useState("All Users");
+  const Button = (category: string): unknown => {
+    return (
+      <button
+        type="button"
+        className={
+          title === category
+            ? "bg-gray-200 py-3 px-8 rounded-full font-bold tracking-wide"
+            : "py-3 px-8 rounded-full text-gray-500 tracking-wide"
+        }
+        onClick={() => {
+          setTitle(category);
+        }}
+      >
+        {category}
+      </button>
+    );
+  };
+  return (
+    <div>
+      <div className="flex flex-row justify-between text-sm text-center">
+        {Button("All Roles")}
+        {Button("Admin")}
+        {Button("Players")}
+        {Button("Mentors")}
+        {Button("Donors")}
+        {Button("Parents")}
+      </div>
```

Comment:
can you add a TODO comment that the dashboard components corresponding to each tab should go here?